fairscale/utils/parallel.py

```diff
@@ -58,7 +60,14 @@ def enable_pytorch_sync_bn(module: torch.nn.Module) -> None:
             layer._specify_ddp_gpu_num(1)  # type: ignore
 
 
-def get_process_group_cached(ranks: Optional[Sequence[int]] = None) -> ProcessGroup:
+class ProcessGroupName(str, Enum):
+    main = "main"
```

Comment:
nit: default

Reply:
or name it all_gather_and_compute?

Reply:
I will switch to default because if the parameter for this feature is turned off, all the communications will fall back the this group.

Reply:
done